src/react/components/common/tagInput/tableTagConfig.scss

```diff
@@ -0,0 +1,25 @@
+@import "../../../../assets/sass/theme.scss";
+@import "./tagInputSize.scss";
+
+.detailsListRows {
+}
+
+.zzpppzz2 {
+    width: 400px;
+}
+
+.zzpppzz {
```

Comment:
add a meaningful name than "zzpppzz"?

Reply:
This is a draft pull request. No need to review yet